packages/contracts/util/exchange_wrapper.ts

```diff
@@ -23,14 +25,14 @@ export class ExchangeWrapper {
     ): Promise<TransactionReceiptWithDecodedLogs> {
         const shouldThrowOnInsufficientBalanceOrAllowance = !!opts.shouldThrowOnInsufficientBalanceOrAllowance;
         const params = order.createFill(shouldThrowOnInsufficientBalanceOrAllowance, opts.fillTakerTokenAmount);
-        const txHash = await this._exchange.fillOrder(
+        const txHash = await this._exchange.fillOrder.sendTransactionAsync(
             params.orderAddresses,
             params.orderValues,
             params.fillTakerTokenAmount,
             params.shouldThrowOnInsufficientBalanceOrAllowance,
-            params.v,
-            params.r,
-            params.s,
+            params.v as number,
+            params.r as string,
+            params.s as string,
```

Comment:
Why are these not automatically detected as being `number` and `string` types?

Reply:
In 0x.js we have SignedOrder and Order. In contracts we have Order with `v: number|undefined`. This is a topic for a future refactor. For now I just show the compiler that I'm sure that this order is signed at the moment.

Reply:
ok. Please make sure to add a task for this refactor.